### PR TITLE
72 run equibind on plex

### PR DESCRIPTION
**Pull request by NiklasTR**

### CODE REVIEW

File: plex/bacalhau.go [hunk 1]
```diff
@@ -6,11 +6,16 @@ import (
 	"strings"
 )
 
-func InstructionToBacalhauCmd(cid, container, cmd string) string {
-	// TODO allow overrides for gpu memory and network flags
-	return `bacalhau docker run --network full --gpu 1 --memory 12gb -i ` + fmt.Sprintf(cid) + ` ` + fmt.Sprintf(container) + ` -- ` + fmt.Sprintf(cmd)
+func InstructionToBacalhauCmd(cid, container, cmd string, gpu string) string {
+	// TODO allow boolean overrides for gpu memory and network flags
+	gpuFlag := ""
+	if gpu != "false"{
```
Comment (thetechnocrat-dev) [inline plex/bacalhau.go:12]: 🙈

**Review by thetechnocrat-dev — APPROVED** [review]
`/plex -app equibind -input-dir ../af_1jsn -gpu false`

Worked like a charm docking SAa2,3 to Avian H5 Haemagglutinin. Kinda close, but not nearly as close to the experimental structure as DiffDock so a good example. Ran in 7 seconds.


Screen Shot 2023-02-16 at 10 35 37 AM

File: plex/bacalhau.go (hunk 1 above)
Comment (NiklasTR) [reply plex/bacalhau.go:12]: don't hate me

Comment (NiklasTR) [reply plex/bacalhau.go:12]: everything is a string at the end of the day

**Pull request merged**